src/keyboard.js

```diff
@@ -82,14 +83,34 @@ export function getCode( key ) {
  */
 export function parseKeystroke( keystroke ) {
 	if ( typeof keystroke == 'string' ) {
-		keystroke = keystroke.split( /\s*\+\s*/ );
+		keystroke = splitKeystrokeText( keystroke );
 	}
 
 	return keystroke
 		.map( key => ( typeof key == 'string' ) ? getCode( key ) : key )
 		.reduce( ( key, sum ) => sum + key, 0 );
 }
 
+/**
+ * It translates any keystroke string text like `"CTRL+A"` to an
+ * environment–specific keystroke, i.e. `"⌘A"` on Mac OSX.
+ *
+ * @method utils.keyboard.getEnvKeystrokeText
+ * @param {String} keystroke Keystroke text.
+ * @returns {String} Keystroke text specific for the environment.
+ */
+export function getEnvKeystrokeText( keystroke ) {
+	const split = splitKeystrokeText( keystroke );
+
+	if ( env.mac ) {
+		if ( split[ 0 ].toLowerCase() == 'ctrl' ) {
```

Comment:
BTW. Shift and Alt should also be handled by this function.

BTW2. The standard for defining keystrokes in CKE4 is: 

![screen shot 2016-09-06 at 16 37 50](https://cloud.githubusercontent.com/assets/156149/18277873/49388fa6-7450-11e6-9fc3-90f26b1f06db.png)